### ISSUE TITLE
`statistics` module required in Python 2.7

### ISSUE THREAD

**Issue by matiasgarciaisaia** [sarahsharp/foss-heartbeat#31]
The README.md doesn't state anything about the Python version required for the project to run.

When running using Python 2.7, there's an extra dependency to add:

```
$ python --version
Python 2.7.12
$ python ghstats.py crystal crystal-lang docs/
Traceback (most recent call last):
  File "ghstats.py", line 41, in <module>
    import statistics
ImportError: No module named statistics
```

Doing `pip install statistics` will do the trick. This should probably work on Python 3, since [`statistics` package](https://pypi.python.org/pypi/statistics/) is _A Python 2.* port of 3.4 Statistics Module_.

This _may be_ a <3.4 issue, though - I'm not a Pythoneer at all.

It would be to add a notice in the README, or maybe there's a way to list it in the requirements.txt that won't conflict Python 3.4 installations - I don't know how you'd do that.

**Comment by dimpase** [comment]
indeed, I can confirm that on python 2.7 `statistics` should be in `requirements.txt`.

**Comment by matiasgarciaisaia** [comment]
Seems to have been solved in #36 - thanks @mureinik for picking this up!
